lib/nodeshift-config.js

```diff
@@ -60,6 +60,8 @@ async function setup (options = {}) {
     throw new Error('"name" in package.json can only consist lower-case letters, numbers, and dashes. It must start with a letter and can\'t end with a -.');
   }
 
+  options.outputImageStreamName = options.outputImageStreamName || projectPackage.name;
+  options.outputImageStreamTag = 'latest';
```

Comment:
Shouldn't this be `options.outputImageStreamTag || `latest`?

Reply:
same answer as above

Reply:
looks like i forgot to write a test that would have caught that

Reply:
updated with the new commits